### PR TITLE
Remove the DSA params from the noderefs

### PR DESCRIPTION
**Pull request by nextgens**
This is the second part of #440 

DO NOT MERGE before 1471 is mandatory!

### CODE REVIEW

File: src/freenet/node/NodeCrypto.java [hunk 1]
```diff
@@ -495,18 +443,11 @@ private String ecdsaSignRef(String mySignedReference) throws NodeInitException {
 
 	void addPrivateFields(SimpleFieldSet fs) {
 	    // Let's not add it twice
-	    fs.removeSubset("ecdsa");
-	    fs.put("ecdsa", ecdsaP256.asFieldSet(true));
```
Comment (unixninja92) [inline src/freenet/node/NodeCrypto.java]: Why did you delete these tabs?

Comment (nextgens) [inline src/freenet/node/NodeCrypto.java]: I've replaced the four spaces with two tabs... to be consistent with the rest of the file
